### PR TITLE
[TAO-8934] WTP-1401 - TCiaB : Magnifying glass does not magnify contents correctly.

### PR DESCRIPTION
**Pull request by AntonTsymuk**
Related to : https://oat-sa.atlassian.net/browse/TAO-8934
 
Capture scrolling changes constantly, but not only when magnifier is active
  
#### How to test

- prepare a test item which does not fit browser screen size
- prepare a test with the test item and magnifying test taker tool enabled
- prepare a delivery for the test and execute it as a test taker
- go to the test item and scroll down to the bottom of the page
- activate the magnifying tool and make sure it magnifies correct area of the page

### CODE REVIEW

File: src/plugins/tools/magnifier/magnifierPanel.js [hunk 1]
```diff
@@ -93,6 +103,39 @@ var dynamicComponentDefaultConfig = {
...
+            id: scrollId,
+            scrollTop: scrollingTop,
+            scrollLeft: scrollLeft
+        });
+    }
+
+    componentScrollCallback && componentScrollCallback(scrollData);
+}, scrollingDelay);
+
+window.addEventListener('scroll', scrollingListenerCallback, true);
```
Comment (bziondik) [inline src/plugins/tools/magnifier/magnifierPanel.js]: @jsconan  I am not sure about listening scrolls every time even user don't use the plugin.
Maybe there is some way to calculate scroll right before the plugin will show?

Comment (jsconan) [reply src/plugins/tools/magnifier/magnifierPanel.js]: Indeed, you are right. Even with the throttle applied on the callback, the event should only be applied when needed.

Comment (AntonTsymuk) [reply src/plugins/tools/magnifier/magnifierPanel.js]: yes, but i did not find another option how to restore scroll position when plugin is activated

Comment (jsconan) [reply src/plugins/tools/magnifier/magnifierPanel.js]: The issue is the scroll position is not accurate when the plugin is activated. Then why not properly initialize the callback context, without moving it?

Something like:
```
...
    let scrollPosition = getCurrentScrollPosition();
    var scrollingListenerCallback = _.throttle(function(event) {
        var $target = $(event.target);
        var scrollingTop = event.target.scrollTop + scrollPosition.top;
        var scrollLeft = event.target.scrollLeft + scrollPosition.left;
        ...
    }, scrollingDelay);
...
    /**
     * Initializes the listener for scrolling event and transfer the scrolling
     */
    function setScrollingListener() {
        scrollPosition = getCurrentScrollPosition();
        window.addEventListener('scroll', scrollingListenerCallback, true);
    }

    /**
     * Stops the listener for scrolling event
     */
    function removeScrollingListener() {
        window.removeEventListener('scroll', scrollingListenerCallback, true);
    }
...
```

**Pull request merged**